controller/deployments_osioclient.go

```diff
@@ -80,7 +80,7 @@ func CreateOSIOClient(witclient WitClient, responseReader ResponseReader) Opensh
 // if userService is nil, will fetch the user services under the hood
 func (osioclient *OSIOClient) GetNamespaceByType(ctx context.Context, userService *app.UserService, namespaceType string) (*app.NamespaceAttributes, error) {
 	if userService == nil {
-		us, err := osioclient.GetUserServices(ctx)
+		us, err := osioclient.GetUserServices(goasupport.ForwardContextRequestID(ctx))
```

Comment:
I think this call is redundant.

Reply:
Ah, yes, you're right. Fixed in https://github.com/fabric8-services/fabric8-wit/pull/2062/commits/8a6dfcd36e5eadbe188a21be4f2fca343a2c44f4